patches/v8/revert_fastapi_promote_deprecation_of_fastapitypedarray.patch

```diff
@@ -0,0 +1,25 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: David Sanders <dsanders11@ucsbalum.com>
+Date: Wed, 30 Oct 2024 21:40:17 -0700
+Subject: Revert "[fastapi] Promote deprecation of FastApiTypedArray"
+
+This reverts commit 22b9e934bbbc2f026ed2c3a97a72d9f3ec890a68.
+
+Revert this until Node.js decides how to proceed and then pick up their fix.
+
+Refs: https://github.com/nodejs/node/issues/55452
+Refs: https://chromium-review.googlesource.com/c/v8/v8/+/5956408
```

Comment:
Not sure if you are aware but https://github.com/v8/node is useful to backport api breakage changes when revert is hard to maintain, the repo is maintained for node ci integration in the V8 repo. Current active branch https://github.com/v8/node/commits/node-ci-2024-10-02/

The perf implication for this particular issue is already addressed in upstream https://issues.chromium.org/u/0/issues/356158110 but maybe we are not impacted in the first since our build of Node.js aligns with Chrome.

Reply:
Thanks for the repo link, I wasn't aware of that one. I was told about https://github.com/nodejs/node-v8 in the past couple weeks so I checked there but didn't see anything related to it. Good to know, I'll check that one going forward.

Reply:
Yeah `node-v8` should be the primary choice since the commits there eventually land in the branches of `nodejs/node`. But when hit with breaking change that is kind of hard to revert and there is no relevant commit yet in `node-v8` it is good to pick changes from `v8/node` since the build configurations align with how we build Node.js.